docs/embedded.adoc

```diff
@@ -484,18 +484,26 @@ Red Hat Data Grid supports eviction of entries, such that you do not run out of
 
 [NOTE]
 ====
-Eviction occurs on a local basis, and is not cluster-wide. Each node runs an eviction thread to analyse the contents of its in-memory container and decide what to evict. Eviction does not take into account the amount of free memory in the JVM as threshold to starts evicting entries. You have to set size attribute of the eviction element to be greater than zero in order for eviction to be turned on. If size is too large you can run out of memory. The size attribute will probably take some tuning in each use case.
+Eviction occurs on a local basis, and is not cluster-wide. Each instance will analyze on adding a new entry whether the threshold for evction is reached and decide what to evict. Eviction does not take into account the amount of free memory in the JVM as threshold to starts evicting entries. You have to set size attribute of the eviction element to be greater than zero in order for eviction to be turned on. If size is too large you can run out of memory. The size attribute will probably take some tuning in each use case.
+====
+
+[NOTE]
+====
+Eviction is not recommended to use without a persistence as it will cause inconsistencies between the clustered instances!
 ====
 
 === Difference between Eviction and Expiration
 
-Both Eviction and Expiration are means of cleaning the cache of unused entries and thus guarding the heap against OutOfMemory exceptions, so now a brief explanation of the difference.
+Both Eviction and Expiration are means of cleaning the cache of unused entries and thus guarding the heap against OutOfMemory exceptions, but eviction is primary to control the data in memory and expirationis to control the lifecycle of entries.
+So now a brief explanation of the difference.
 
-- With eviction you set maximal number of entries you want to keep in the cache and if this limit is exceeded, some candidates are found to be removed according to a choosen eviction strategy (LRU, LIRS, etc…​). Eviction can be setup to work with passivation, which is eviction to a cache store.
+- With eviction you set maximal number of entries you want to keep in the memory and if this limit is exceeded, some candidates are found to not longer kept in memory according to the eviction strategy (LRU, LIRS, etc…​).
+  - TinyLFU with caffeine or LRU if Off-Heap memory is enabled
+  - Eviction can be set up with passivation in that case the entry is only persisted if evicted from memory
```

Comment:
We haven't mentioned caffeine uptill now, and the user might get confused about what it is. I think we need to add some sort description with a CTA to docs.